jade/wsgi/routes/v1/judgements.py

```diff
@@ -2,49 +2,78 @@
 
 import flask
 
-from func_tools import wraps
+from ... import responses, util
+from .... import types
 
-from ... import responses
 
+def configure(bp, config, trusted_clients, centralauth, state, stream):
```

Comment:
Nesting these route functions under a top-level function rather than a class will make testing difficult, won't it?  Aren't these inner functions inaccessible?

Reply:
They are inaccessible at first, but they become part of `bp`.  This actually provides a nice framework for testing because the `bp` object is a general routing abstraction.  Rather than using global variables, shared resources are enclosed in the configure() function.  This allows us to, for example, safely pass an arbitrary configuration in testing or when constructing the routes to use.  We can also easily include mocks for trusted_clients, centralauth, etc.

Reply:
Okay, thanks it sounds fine in that case.  The function is pretty much equivalent to a class, which means we can safely have slightly different opinions about the best way to encapsulate, since they'll behave in roughly the same way :)

Reply:
Oh interesting.  I wonder if this *would* be better as a class.  I think bp takes the place of a class.  But we could make the functions themselves available publicly/directly.